components/about-us/introduction.tsx

```diff
@@ -104,6 +106,18 @@ export default function IntroductionAboutUsComponent() {
           </Stack>
         </Box>
       </Stack>
+      <Box
+        position='absolute'
+        top='250px'
```

Comment:
I see you used relative units below.  can we use relative units throughout.  The mobile view looks a little off and i think its just not changing size properly.
![image](https://user-images.githubusercontent.com/96752460/168214772-49287251-9d6c-457d-b42a-cf7585e564b5.png)
vs the figma
![image](https://user-images.githubusercontent.com/96752460/168214844-005a3d2d-5ba5-4b0b-8463-e0ca99be3f3b.png)